pkg/controller/common/stackmon/sidecar.go

```diff
@@ -125,3 +101,24 @@ func NewBeatSidecar(ctx context.Context, client k8s.Client, beatName string, ima
 		Volumes:      podVolumes,
 	}, nil
 }
+
+func CAVolume(
```

Comment:
nit: for new readers, let's maybe add a godoc to explain which CA we are referring to here as there might be 2 CAs involved in the process (the one to connect to monitored component and the one from the association).